web/sites/settings.local.php

```diff
@@ -0,0 +1,161 @@
+<?php
```

Comment:
I don't think we want this file committed without adding something like this into the settings.php file too. We wouldn't want these to end up running on production.

```if (isset($_ENV['LANDO_INFO'])) {
  $lando_settings = __DIR__ . "/settings.local.php";
  if (file_exists($lando_settings)) {
    include $lando_settings;
  }
}
```

Reply:
Just a heads up... this file is even in the wrong directory for settings.local.php to take affect. I should be in the web/sites/default folder instead of the web/sites folder.